### PR TITLE
fix #26

### LINKED ISSUE #26: add_phylopic_base issue
user asks if the line `graphics::par(usr = c(0, 1, 0, 1))`  https://github.com/sckott/rphylopic/blob/master/R/add_phylopic_base.r#L46 should be changed/removed 

says 

> So, these settings limit the plotting area. I have removed it and been able to place the silhouettes in the top of the observation points.

any ideas @dill ?

### PR DESCRIPTION
**Pull request by dill**
## Description

Better handling of image scale, avoids use of `par(usr...` bits.

## Related Issue

fix #26 

## Example

```r
library(rphylopic)
add_phylopic_base2 <- function (img, x = NULL, y = NULL, ysize = NULL, alpha = 0.2, 
                                   color = NULL) 
{
  img <- rphylopic:::recolor_phylopic(img, alpha, color)
  dims <- dim(img)[1:2]
  AR <- dims[1]/dims[2]
  # graphics::par(usr = c(0, 1, 0, 1))
  graphics::rasterImage(img, x - (ysize/2), y - (AR * ysize/2), 
                        x + (ysize/2), y + (AR * ysize/2), interpolate = TRUE)
}

## My sampled species
z <- image_data("7f2cbb42-12b1-4481-8ac0-705eb7363c74", size = "128")[[1]]

## Coordinates

set.seed(999)
X <- runif(10, 0, 2)
Y <- runif(10, 0, 2)

par(mfrow=c(2,1))

## Plot using the original formula
plot(X, Y, pch=3, col="red", xlim=c(0,2), ylim=c(0,2), main="Original function")
add_phylopic_base(z, x = X, y = Y, ysize = 0.4) 

## Plot using the modifyed funtion. Trout appear on the top of the observations
plot(X, Y, pch=3, col="red", xlim=c(0,2), ylim=c(0,2), main="Modified function")
add_phylopic_base2(z, x = X, y = Y, ysize = 0.4) 
```

Screen Shot 2020-06-02 at 18 06 15

"original function" above is now what is in `rphylopic` :)

### CODE REVIEW

File: R/add_phylopic_base.r [hunk 1]
```diff
@@ -32,28 +32,33 @@
...
 #' }
 #' 
-#' # plot a stand alone image of the silhouette
-#' plot_phylopic_base(cat)
+#' # Example using a cat background
+#' # setup plot area
+#' plot(posx, posy, type="n", main="A cat herd, on top of a cat",
+#'      xlim=c(0,1), ylim=c(0,1))
+#' # get a higher-resolution cat
+#' cat_hires <- image_data("23cd6aa4-9587-4a2e-8e26-de42885004c9", size = 128)[[1]]
```
Comment (sckott) [inline R/add_phylopic_base.r:40]: seems like this isn't any higher res? 128 and 128. maybe use 512?

**Comment by dill** [comment]
oh whoops, really must start wearing my glasses when writing PRs...

**Comment by sckott** [comment]
👌

**Comment by sckott** [comment]
purrfct

**Pull request merged**